
<launch>
  <node name="ak70_odom" pkg="ak70_odom" type="cmdvel_to_ak70.py" output="screen">
    
    <param name="can_interface" value="can0"/>
    <param name="left_id"  value="0x02"/>
    <param name="right_id" value="0x01"/>
    <param name="wheel_radius" value="0.1"/>
    <param name="wheel_base"   value="0.339"/>
    <param name="left_dir"  value="1"/>
    <param name="right_dir" value="-1"/>

    <param name="erpm_limit" value="5000.0" />
    <param name="acc_limit"  value="5000.0" />
    <param name="rate_hz"    value="100" />
    
    <param name="linear_correction"  value="1.0" />
    <param name="angular_correction" value="1.0" />
    <param name="publish_joint_states" value="true"/>
    <param name="/use_sim_time" value="false"/>
  </node>
</launch>
